src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -878,7 +866,7 @@ class ShenandoahScanRemembered: public CHeapObj<mtGC> {
 // A ShenandoahRegionChunk represents a contiguous interval of a ShenandoahHeapRegion, typically representing
 // work to be done by a worker thread.
 struct ShenandoahRegionChunk {
-  ShenandoahHeapRegion *_r;      // The region of which this represents a chunk
+  ShenandoahHeapRegion* _r;      // The region of which this represents a chunk
   size_t _chunk_offset;          // HeapWordSize offset
   size_t _chunk_size;            // HeapWordSize qty
 };
```

Comment:
At line 880 below:

"other worker will threads repeatedly ..." should be "other worker threads will repeatedly".